lib/image-diff.js

```diff
@@ -95,6 +96,7 @@ ImageDiff.prototype = {
     var actualPath = options.actualImage;
     var expectedPath = options.expectedImage;
     var diffPath = options.diffImage;
+    var shadow = options.shadow ? ['', ''] : ['-compose','Src'];
```

Comment:
Instead of generating parameters here, can we pass through `shadow` to `createDiff` like `diffPath`/`diffImage`?